src/components/form/Lookup/LookupField.tsx

```diff
@@ -64,20 +71,31 @@ export function LookupField<
   T,
   Multiple extends boolean | undefined,
   DisableClearable extends boolean | undefined,
-  FreeSolo extends boolean | undefined
+  FreeSolo extends boolean | undefined,
+  CreateFormValues = never
 >({
   name: nameProp,
   multiple,
+  freeSolo,
   defaultValue,
   disabled: disabledProp,
   useLookup,
   ChipProps,
   helperText,
   label,
   required,
+  CreateDialogForm,
+  getOptionLabel,
```

Comment:
Do you prefer doing` props.getOptionLabe`l to passing it directly? And what do you mean recheck that all changes are necessary?

Reply:
That's fine. I think calling it via `props.` helps to represent that this isn't our prop, but a prop of the underlying component we are just also using. It's also less code because you don't have to pass it down explicitly.

Reply:
True